docs/admin-guides/deployment.md

```diff
@@ -24,7 +24,7 @@ You can choose to download the stable release package from [download page](../..
 ## Download the distribution
 
 All released package can be downloaded from [download page](../../../download/).
-You can download amoro-x.y.z-bin.zip (x.y.z is the release number), and you can also download the runtime packages for each engine version according to the engine you are using.
+You can download apache-amoro-x.y.z-bin.tar.gz (x.y.z is the release number), and you can also download the runtime packages for each engine version according to the engine you are using.
 Unzip it to create the amoro-x.y.z directory in the same directory, and then go to the amoro-x.y.z directory.
```

Comment:
apache-amoro-x.y.z-bin directory in the same directory, and then go to the apache-amoro-x.y.z-bin directory.

Reply:
The directory of the unzip result remains `amoro-x.y.z`.